### DISCUSSION TITLE
main.mjs location

### DISCUSSION THREAD

**Discussion by ChopperDan** [felddy/foundryvtt-docker#272] [Q&A] [answered]
Where is "main.mjs" located? I'm trying to modify for an art browser backend...
Thanks!

**Comment by felddy** [comment, marked as answer]
There is no `main.mjs` in the source code for this container.  That file is part of FoundryVTT.

Once the container starts and a distribution of FoundryVTT is successfully installed, `main.mjs` will be located at:

`"${FOUNDRY_HOME}/resources/app/main.mjs"`

**Comment by ChopperDan** [comment]
I was able to find it, and enable what I needed! Thank you.
